base/docs/helpdb/Base.jl

```diff
@@ -784,12 +784,12 @@ Equivalent to `stat(file).mode`
 filemode
 
 """
-    print_joined(io, items, delim, [last])
+    join(io, items, delim, [last])
```

Comment:
Yeah. The brackets are just for presentation, the part outside controls which method this particular docstring gets attached to, AIUI. I think we may eventually want to move from the bracket convention to strictly using Julia syntax in docstrings, but that's a decision for another day.